src/main/helpers/ps.ts

```diff
@@ -1,12 +1,32 @@
 import psList from "ps-list";
-import { tasklist } from "tasklist";
+import path from "node:path";
```

Comment:
Could you please uninstall `tasklist`? This library will no longer be necessary.